anypoint-mq/2.x/modules/ROOT/pages/anypoint-mq-ack.adoc

```diff
@@ -8,8 +8,14 @@ The Anypoint MQ connector provides three acknowledgment configurations: Immediat
 == Immediate Acknowledgment
 
 Use the `IMMEDIATE` acknowledgment mode to acknowledge messages as your app consumes them, before processing them.
+When a message is acknowledged, Anypoint MQ removes it from the queue.
```

Comment:
I see similar messaging in previous section `Acknowledging a message (ACK) informs the broker that the message has been processed and must be removed from the queue to prevent redelivery. Not acknowledging a message (NACK) tells the broker to not process the message and to return the message to the queue for redelivery to any available consumer.` 

May be we dont need this new one here ?

Reply:
@kparadkar-ms - Well, the first section is really the intro to the whole topic, which some people skip to get to the thing they're looking for, so I sort of think we need it here too. I suppose we could remove it from the intro.

Reply:
Oh, OK. I see what you mean. Deleted.